react-components/src/hooks/use3dScenes.tsx

```diff
@@ -58,25 +61,36 @@ export const use3dScenes = (
           const spaceMap = acc[space];
 
           if (spaceMap[externalId] === undefined) {
-            spaceMap[externalId] = [];
+            spaceMap[externalId] = { name: '', modelOptions: [] };
           }
           const externalIdMap = spaceMap[externalId];
-          externalIdMap.push(newModel);
+          externalIdMap.modelOptions.push(newModel);
 
           return acc;
         },
         // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
What is this needed for?

Reply:
We use the `modelOptions` in use3dScenes for finding mapped objects for a given scene in other applications